electionleaflets/settings/testing.py

```diff
@@ -1,11 +1,16 @@
 import os
+from tempfile import mktemp
 
 from .base import *  # noqa: F403
+os.environ["DJANGO_ALLOW_ASYNC_UNSAFE"] = "true"
 
-DEFAULT_FILE_STORAGE = "django.core.files.storage.FileSystemStorage"
-STATICFILES_STORAGE = "django.core.files.storage.FileSystemStorage"
-TEST_RUNNER = "django.test.runner.DiscoverRunner"
+DEFAULT_FILE_STORAGE = "electionleaflets.storages.TempUploadLocalMediaStorage"
+STATICFILES_STORAGE = "electionleaflets.storages.StaticStorage"
 
-MEDIA_ROOT = root("test_media",)  # noqa: F405
+MEDIA_ROOT = mktemp()
```

Comment:
Minor, but this creates a new tempdir for every test run and we never clean it up
It would be nice if we could find a way to remove this dir in a post hook after the test suite runs.
Also if we are not using `test_media` any more, we can clean up the gitignore entries relating to that directory.

Reply:
Fixed in https://github.com/DemocracyClub/electionleaflets/pull/257/commits/16b30c9ebef83116d4f96908b3bd594ff6097f84

Reply:
nice :+1: